Fortinet/fortiweb/ingest/parser.yml

```
@@ -30,10 +32,17 @@ stages:
           network.protocol: "{{parsed_event.message.proto}}"
           rule.ruleset: "{{parsed_event.message.policy}}"
           source.geo.name: "{{parsed_event.message.srccountry}}"
-          source.ip: "{{parsed_event.message.src}}"
+          source.ip: "{{parsed_event.message.src or parsed_event.message.src_ip}}"
           source.port: "{{parsed_event.message.src_port}}"
           tls.cipher: "{{parsed_event.message.cipher_suite}}"
+          url.domain: "{{parsed_event.message.http_host}}"
           url.path: "{{parsed_event.message.http_url}}"
           url.username: "{{parsed_event.message.user_name}}"
           user.name: "{{parsed_event.message.user}}"
           user_agent.original: "{{parsed_event.message.http_agent}}"
+          user.id: "{{parsed_event.message.user_id}}"
+
+      - set:
+          user.name: "{{parsed_event.message.user_name.split('@')[0]}}"
+          user.domain: "{{parsed_event.message.user_name.split('@')[1]}}"
```

I suggest adding `user.email` also
```suggestion
          user.domain: "{{parsed_event.message.user_name.split('@')[1]}}"
          user.email: "{{parsed_event.message.user_name}}"
```